wrappers/MATLAB/refpropm.m

```diff
@@ -184,30 +186,35 @@
     numComponents = nargin - nc_base - 1;
 end
 
-fluidType = [];
-for i = 1:numComponents
-    fluidType = [fluidType char(varargin(5+i))];
-end
+fluidType = [varargin{5+(1:numComponents)}];
 
 % Load DLL
-RefpropLoadedState = getappdata(0, 'RefpropLoadedState');
-if ~libisloaded(libName)
+loaded = libisloaded(libName);
+if ~loaded || isempty(RefpropLoadedState)
     FluidDir = 'FLUIDS/';
     switch computer
         case {'GLNXA64', 'GLNX86', 'MACI', 'MACI64', 'SOL64'}
-            BasePath = strcat(getenv('HOME'),'/refprop/');
+            if ~isempty(getenv('REFPROP_BASE'))
+                BasePath = getenv('REFPROP_BASE');
+            else
+                BasePath = strcat('/usr/local/refprop/');
+                if exist(BasePath, 'dir') ~= 7

```

Comment:
Removing this change is OK for me. It was mainly to better fit my own needs.
If keeping it, I will edit the MATLAB README file to document this change.